variables.tf

```diff
@@ -37,7 +37,7 @@ variable "helm_release_name" {
 }
 
 variable "helm_repo_name" {
-  default = "stable"
+  default = "eks-stable"
```

Comment:
@remeq Why do we need to change the default repo name?

Reply:
It is rewriting some of the existing helm repos and then helm is failing to find charts